TopologicalSort/topological_sort.py

```diff
@@ -0,0 +1,61 @@
+# """
+# Topological Sort
+# Difficulty: Hard
+# Given a list of tasks and a list of dependencies return
+# a list of tasks in valid order. If no such order exists
+# return an empty array.
+#
+# tasks = [1,2,3,4]
+# deps = [
+#     [2, 3],
+#     [2, 4],
+#     [4, 3],
+#     [1, 3],
+#     [1, 4],
+# ]
+#
+# topological_sort(tasks, deps) = [1,2,4,3] or [2,1,4,3]
+#
+# Author: BrannanC
+# """
+from collections import defaultdict
+
+
+def topological_sort(j, deps):
+    jobs = j[::1]
```

Comment:
Is this just to copy j so you don't mutate it?

Reply:
yes